frontend/src/container/MetricsApplication/Tabs/DBCall.tsx

```diff
@@ -16,16 +21,26 @@ const DBCall = ({ getWidget }: DBCallProps): JSX.Element => {
 					<Card>
 						<GraphTitle>Database Calls RPS</GraphTitle>
 						<GraphContainer>
-							<FullView
+							<Graph
 								name="database_call_rps"
-								noDataGraph
-								fullViewOptions={false}
-								widget={getWidget([
-									{
-										query: `sum(rate(signoz_db_latency_count{service_name="${servicename}"}[1m])) by (db_system)`,
-										legend: '{{db_system}}',
-									},
-								])}
+								type="line"
+								data={{
+									datasets: dbRpsEndpoints.map((data, index) => {
```

Comment:
can we make a component component in which passing the state would reflect the same graph data